composer/trainer/trainer.py

```diff
@@ -791,25 +793,16 @@ def __init__(
                 DeprecationWarning((f"Using the 'grad_clip_norm' field in Trainer is deprecated. Please use"
                                     'the GradientClipping Algorithm in composer.algorithms.gradient_clipping.')))
 
-            print_warning = False
             if algorithms is not None:
```

Comment:
With the above change, I think `algorithms` will always be a list now, and then this check can be removed?